### PR TITLE
BONNIE-1455: hybrid measure support to display SDE

### PR DESCRIPTION
**Pull request by c-monkey**
Tests for this back-end flag are in the accompanying Bonnie PR https://github.com/projecttacoma/bonnie/pull/1013 in `test/functional/measures_controller_test.rb`

Pull requests into Bonnie Bundler require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [X] This pull request describes why these changes were made.
- [X] This PR is into the correct branch.
- [X] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1455
- [x] JIRA ticket links to this PR
- [X] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [X] Tests are included and test edge cases -- see 
- [X] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [X] Code coverage has not gone down and all code touched or added is covered.
    * In rare situations, this may not be possible or applicable to a PR. In those situations:
        1. Note why this could not be done or is not applicable here:
        2. Add TODOs in the code noting that it requires a test
        3. Add a JIRA task to add the test and link it here:
- [X] Automated regression test(s) pass
    * No measures with differences
    * Several measures had covered clauses now uncovered (SDE Ethnicity, Race, Sex, and Payer):
        * rachel@oregonurology.com - CMS771v1
        * epecqmmpr@gmail.com - CMS460v0
        * hmdmnewmeasures@gmail.com - CMS812v2
        * hmdmtjc2@gmail.com - CMS334v1 and CMS334v1

If JIRA tests were used to supplement or replace automated tests:
- [X] JIRA test links: N/A
- [X] Justification for using JIRA tests: N/A
- [X] JIRA tests have been added to sprint N/A

**Reviewer 1:**

Name: @mayerm94 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [X] JIRA tests have been run and pass N/A
- [X] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

**Reviewer 2:**

Name: @jbradl11 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [X] JIRA tests have been run and pass N/A
- [X] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

### CODE REVIEW

File: lib/models/cql_measure.rb [hunk 1]
```diff
@@ -20,6 +20,7 @@ class CqlMeasure
 
   field :episode_of_care, type: Boolean
   field :continuous_variable, type: Boolean
+  field :hybrid, type: Boolean
```
Comment (mayerm94) [inline lib/models/cql_measure.rb]: I am hesitant about using the wording 'hybrid' since it only means to include SDEs. Could we change this to 'calculate_SDEs' or something along those lines? This would make the code more easy to understand (don't need the domain knowledge of what a hybrid measure is)

**Review by jbradl11 — APPROVED** [review]
Simple change. Looks good.

**Review by mayerm94 — APPROVED** [review]
